### PR TITLE
Santa Hat Sprite Tweak

### PR DESCRIPTION
**Pull request by Ionward**
**What does this PR do:**
This touches up the Santa hat sprite, adding a white fuzzy brim, making it look less like a hood, and adding a second variant without the beard, toggleable by using it in your hand, like the ushanka.

**Images of sprite/map changes (IF APPLICABLE):**
From:
santahat1
santahat2
santahat3

To:
down
side
up
item icons
inhand beard

Including a message that appears when you use the Santa hat:
text example

**Changelog:**
:cl: Ionward
tweak: updated santa hat sprite, use hat to toggle beard
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Ho ho holy shit that looks great, nicely done

**Comment by Ty-Omaha** [comment]
hot

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -99,8 +99,22 @@
...
 	icon_state = "santahat"
+	species_fit = list("Grey, Drask")
+	sprite_sheets = list(
+		"Grey" = 'icons/mob/species/Grey/head.dmi',
+		"Drask" = 'icons/mob/species/Drask/helmet.dmi'
+		)
 	flags = BLOCKHAIR | STOPSPRESSUREDMAGE
 	flags_cover = HEADCOVERSEYES
+/obj/item/clothing/head/helmet/space/santahat/attack_self(mob/user as mob)
+	if(src.icon_state == "santahat")
```
Comment (Ty-Omaha) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: `src `is implied, you don't need to put it infront - you can just do `if(icon_state == "santahat")` and the same for all the `srcs `below

**Comment by Citinited** [comment]
How does this look on vox? Ho ho skrek

**Comment by Ralta** [comment]
Yes, yes a THOUSAND TIMES YES! :+1:

**Pull request merged**